### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR adds `List of Features` and  `Roadmap` sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,61 @@
...
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
+### Frontend
+
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
```
Comment (Insun35) [inline README.md:72]: I think we can add R1CS here.

Comment (chokobole) [reply README.md:72]: I am thinking to add circom here after r1cs parser is completed!

File: README.md [hunk 2]
```diff
@@ -15,6 +15,61 @@
...
+### Proving Scheme
+
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
+### Frontend
```
Comment (dongchangYoo) [inline README.md:67]: How about "Arithmetization scheme"
[FYI] https://blog.lambdaclass.com/arithmetization-schemes-for-zk-snarks/

Comment (chokobole) [reply README.md:67]: I think this is filled as we cover more frontends and backends support?

**Review by Insun35 — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (chokobole) [reply README.md:67]: They are limited to only 3, right? I want to make a table to be able to grow unlimitedly.

Comment (dongchangYoo) [reply README.md:67]: I mean that it would be preferable to choose one side between (Proving Scheme, Arithmetization Scheme) or (Frontend, Backend).

Comment (chokobole) [reply README.md:67]: Then let's say if we support zokrates, circom, cairo and miden, where do you think they have to go?

File: README.md [hunk 3]
```diff
@@ -15,6 +15,67 @@
 3. Blazing Fast: Tachyon's foremost requirement is speed, and not just any speed, but blazing speed! This entails Tachyon delivering exceptional performance on both CPU and GPU platforms.
 4. GPU Interoperability: Tachyon's code is designed to be compatible with both CPU and GPU in the majority of scenarios.
 
+## List of Features
+
+List of symbols:
+
+- :heavy_check_mark: means the feature is currently supported.
+- 🏗️ means that the feature is partially implemented or is under active construction.
+- :x: means that the feature is not currently supported.
```
Comment (TomTaehoonKim) [inline README.md:24]:
```suggestion
- 🏗️ means the feature is partially implemented or is under active construction.
- :x: means the feature is not currently supported.
```
How about matching the rhymes like the above line like this?

**Review by TomTaehoonKim — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (chokobole) [reply README.md:67]: We decided to use `SNARK` and `STARK` term!

File: README.md (hunk 3 above)
Comment (ashjeong) [reply README.md:24]: oops i also recommended a way to fix this

**Review by dongchangYoo — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -15,6 +15,61 @@
 3. Blazing Fast: Tachyon's foremost requirement is speed, and not just any speed, but blazing speed! This entails Tachyon delivering exceptional performance on both CPU and GPU platforms.
 4. GPU Interoperability: Tachyon's code is designed to be compatible with both CPU and GPU in the majority of scenarios.
 
+## List of Features
+
+Symbol Definition:
```
Comment (ashjeong) [inline README.md:20]: whoops my error
```suggestion
Symbol Definitions:
```

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -15,6 +15,68 @@
...
+|       | CPU                | GPU |
+| ----- | ------------------ | --- |
+| Halo2 | :heavy_check_mark: | :x: |
+
+## Roadmap
+
+- [x] 2024Q1 - Enable producing the [zkEVM](https://github.com/kroma-network/zkevm-circuits) proof.
+- [ ] 2024Q2 - Replace Halo2 with Tachyon in [Kroma](https://kroma.network/) mainnet.
+- [ ] 2024Q3 - Implement Halo2 GPU.
+- [ ] 2024Q4 - Implement Halo2 Folding scheme.
```
Comment (ashjeong) [inline README.md:78]:
```suggestion
- [ ] 2024Q4 - Implement Halo2 Folding Scheme.

**Review by ashjeong — APPROVED** [review]
LGTM

**Pull request merged**